Gemfile.lock

```diff
@@ -325,6 +334,7 @@ DEPENDENCIES
   bootstrap (~> 4.2.1)
   byebug
   capybara (>= 2.15)
+  capybara-screenshot
```

Comment:
:computer: excited for this to be introduced once it is being used, for now though maybe we don't include?